File: python3/2949.py
n = int(input())

x, h, e, a, m = 0, 0, 0, 0, 0

for i in range(n):
    b = input()
    b = b.split(' ')
    if b[1] == 'X':
        x += 1
    elif b[1] == 'H':
        h += 1
    elif b[1] == 'E':
        e += 1
    elif b[1] == 'A':
        a += 1
    elif b[1] == 'M':
        m += 1

print(x, 'Hobbit(s)')
print(h, 'Humano(s)')
print(e, 'Elfo(s)')
print(a, 'Anao(s)')
print(m, 'Mago(s)')

File: uri/cpp/2949.cpp
#include <iostream>

using namespace std;

int main() {

    int n;
    cin >> n;

    int v[5];

    for (int i = 0; i < 5; ++i)
        v[i] = 0;

    string a;
    char b;

    for (int i = 0; i < n; ++i){
        cin >> a >> b;
        if (b == 'X') v[0]++;
        else if (b == 'H') v[1]++;
        else if (b == 'E') v[2]++;
        else if (b == 'A') v[3]++;
        else if (b == 'M') v[4]++;
    }

    cout << v[0] << " Hobbit(s)\n";
    cout << v[1] << " Humano(s)\n";
    cout << v[2] << " Elfo(s)\n";
    cout << v[3] << " Anao(s)\n";
    cout << v[4] << " Mago(s)\n";

    return 0;
}